README.md

```diff
@@ -44,8 +44,8 @@ To join us in these efforts, please feel free to reach out, raise issues or read
 
 Mava currently offers the following building blocks for MARL research:
 
-- 🥑 **Implementations of MARL algorithms**: Implementations of multi-agent PPO systems that follow both the Centralised Training with Decentralised Execution (CTDE) and Decentralised Training with Decentralised Execution (DTDE) MARL paradigms.
-- 🍬 **Environment Wrappers**: Example wrappers for mapping Jumanji environments to an environment that is compatible with Mava. At the moment, we support [Robotic Warehouse][jumanji_rware] and [Level-Based Foraging][jumanji_lbf] with plans to support more environments soon. We have also recently added support for the SMAX environment from [JaxMARL][jaxmarl].
+- 🥑 **Implementations of MARL algorithms**: Implementations of multi-agent PPO systems that follow both the Centralised Training with Decentralised Execution (CTDE) and Decentralised Training with Decentralised Execution (DTDE) MARL paradigms with support for continuous and discrete action space environments.
+- 🍬 **Environment Wrappers**: Example wrappers for mapping Jumanji environments to an environment that is compatible with Mava. At the moment, we support [Robotic Warehouse][jumanji_rware] and [Level-Based Foraging][jumanji_lbf] with plans to support more environments soon. We have also recently added support for the SMAX and MaBrax environments from [JaxMARL][jaxmarl].
```

Comment:
while we're here can we add connector, cleaner, matrax, CVRP and gigastep? Maybe this should become a table of envs we support :thinking:

Reply:
Example:
```python
| Library/Site | Environment Supported | Action Type |
|--------------|-----------------------|-------------|
| Jumanji      | LBF                   | Discrete    |
|              | RWARE                 | Discrete    |
|              | CVRP                  | Discrete    |
| Jaxmarl      | MABRAX                | Continuous  |
|              | SMAX                  | Continuous  |

```

Reply:
If we will mention the different environments I think we need to add a comment that we are working on verifying the performance of mava on them because we may confirm that all works perfectly once we make a full benchmark :pray:

Reply:
**_Update_**: Since the `connector`  env is using a CNN network we may add another column citing the corresponding network for each env and mention that for action space we need to check the corresponding ActionHead as well in the network.yaml file:

```makefile
| Library/Site | Environment Supported | Action Space | Network
|--------------|-----------------------|-------------|-------------
| Jumanji      | LBF                   | Discrete    | Default (mlp)
|              | RWARE                 | Discrete    | Default
|              | Connector             | Discrete    | CNN
| Jaxmarl      | MABRAX                | Continuous  | Default
|              | SMAX                  | Continuous  | Default